apps/aecore/src/aec_db.erl

```diff
@@ -86,7 +92,7 @@
 
 tables(Mode) ->
     [?TAB(aec_blocks)
-   , ?TAB(aec_headers)
+   , ?TAB(aec_headers, [{index, [prev_hash]}])
```

Comment:
In am trying to understand better this index and its test coverage.
Considering that for now Mnesia indexes are delicate (otp_patches), is this index exercised during the [package smoke test](https://github.com/aeternity/epoch/blob/ac3622ea50a37a34084ccdcb320584b0239a7192/.circleci/config.yml#L52)?
If not, is it at least exercised in any one of the aehttp_integration_SUITE/aecore_sync_SUITE/aecore_forking_SUITE? If not, could you please at least add a specific test like in this [(temporary) test (now removed because exercised by API test)](https://github.com/aeternity/epoch/commit/4e60719fb1124d2e28ddf470e0a5c24e91399a66#diff-14dc0eab04e011248c7640cb3c1cf0deL18) ?

Reply:
I refrained from using the plugin indexes, but use a proper, ancient direct index instead. This does not need to be tested more.

It is tested indirectly by the aec_chain_state_tests, though. The index is instrumental for reasoning about the graph structure of the chain.

Reply:
I see that mistyping the index name here (e.g. `prev_hash2`) makes the table creation fail in `make test`. Good.